app/src/main/kotlin/nl/ndat/tvlauncher/data/entity/App.kt

```diff
@@ -15,4 +15,19 @@ data class App(
 
 	val launchIntentUriDefault: String?,
 	val launchIntentUriLeanback: String?,
+
+	@ColumnInfo(defaultValue = "false") val isFavorite: Boolean,
+)
+
+data class AppSystemDetails(
+	var id: String,
+	val displayName: String,
+	val packageName: String,
+	val launchIntentUriDefault: String?,
+	val launchIntentUriLeanback: String?,
+)
+
+data class AppFavorite(
+	var id: String,
+	val isFavorite: Boolean,
 )
```

Comment:
I'm not a fan of these separate views for the same entity. Is this a recommended way of using androidx.room?